jmix-dynattr/dynattr-flowui/src/main/resources/io/jmix/dynattrflowui/view/category/category-detail-view.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!--
+  ~ Copyright 2020 Haulmont.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<view xmlns="http://jmix.io/schema/flowui/view"
+      title="msg://categoryDetailsView.title">
+    <data>
+        <instance id="categoryDc"
+                  class="io.jmix.dynattr.model.Category">
+            <fetchPlan extends="_local">
+                <property name="categoryAttrs" fetchPlan="_local">
+                    <property name="defaultEntity" fetchPlan="_local"/>
+                    <property name="category" fetchPlan="_local"/>
+                </property>
+            </fetchPlan>
+            <loader id="categoryDl"/>
+            <collection id="categoryAttributesDc" property="categoryAttrs"/>
+        </instance>
+        <collection id="categoryAttributesLocationDc" class="io.jmix.dynattr.model.CategoryAttribute"/>
+    </data>
+    <facets>
+        <dataLoadCoordinator auto="true"/>
+    </facets>
+    <actions>
+        <action id="windowCommitAndClose" type="detail_saveClose"/>
+        <action id="windowClose" type="detail_close"/>
+    </actions>
+    <layout spacing="true" expand="tabSheet">
+        <tabSheet id="tabSheet" height="100%" width="100%">
+            <tab id="mainTab" label="msg://mainTab.title">
+                <vbox padding="false" margin="false">
+                    <formLayout id="form" dataContainer="categoryDc">
+                        <responsiveSteps>
```

Comment:
1. steps `minWidth` can be increased. 
2. Empty space between grid and view buttons panel
<img width="519" alt="Screenshot 2023-10-19 at 16 15 45" src="https://github.com/jmix-framework/jmix/assets/6077149/874c035c-2e48-493e-9939-78932659daf6">